activities-examples/package.json

```diff
@@ -24,13 +24,15 @@
   "dependencies": {
     "axios": "0.21.1",
     "node-fetch": "2.x",
-    "temporalio": "0.19.x"
+    "temporalio": "0.19.x",
+    "uuid": "^8.3.2"
```

Comment:
@lorensr move `uuid` to devDependencies?

Reply:
Ah thanks! https://github.com/temporalio/samples-typescript/pull/157